main.js

```diff
@@ -160,4 +160,216 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+console.log("----- opdracht 1a ----");
+const brandsTv = inventory.map((inventory) => {
+return inventory.type
+});
+console.log(brandsTv);
+
+console.log("----- opdracht 1b ----");
+const soldOut = inventory.filter((inventory) => {
+  return inventory.originalStock - inventory.sold === 0;;
+})
+console.log(soldOut);
+
+console.log("----- opdracht 1c ----");
+function hasAmbilight(array) {
+  return array.filter((inventory) => {
+    return inventory.options.ambiLight === true;
+  });
+}
+console.log(hasAmbilight);
+
+console.log("----- opdracht 1d ----");
+function sortPrice(Array) {
+  inventory.sort((a, b) => {
+    if (a.price > b.price) {
+      return 1
+    } else if (a.price < b.price) {
+      return -1
+    } else {
+      return 0
+    }
```

Comment:
Dit kun je nog iets versimpelen
```suggestion
    return a.price - b.price;
```